paper-toast.html

```diff
@@ -40,17 +60,15 @@
         box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.26);
         border-radius: 2px;
         left: 0;
+        bottom: 0;
         margin: 12px;
         font-size: 14px;
         cursor: default;
-        -webkit-transition: visibility 0.3s, -webkit-transform 0.3s, opacity 0.3s;
-        transition: visibility 0.3s, transform 0.3s, opacity 0.3s;
-
-        visibility: hidden;
-        bottom: -100px;
+        -webkit-transition: -webkit-transform 0.3s, opacity 0.3s;
+        transition: transform 0.3s, opacity 0.3s;
```

Comment:
I completely removed `visibility`, as now we are using `opacity`. The effect is the same, both these properties keep the element in the document flow. The advantage of using `opacity` is that we it has a nicer, visible animation.